code/iaas/cluster-logic/src/main/java/io/cattle/iaas/cluster/service/ClusterManager.java

```diff
@@ -0,0 +1,22 @@
+package io.cattle.iaas.cluster.service;
+
+import io.cattle.platform.core.model.Agent;
+import io.cattle.platform.core.model.Host;
+import io.cattle.platform.core.model.Instance;
+import io.cattle.platform.core.model.IpAddress;
+
+public interface ClusterManager {
+
+    Instance createClusterServerInstance(Host cluster);
+
+    Instance getClusterServerInstance(Host cluster);
+
+    Host getClusterForInstance(Instance clusterServerInstance);
```

Comment:
This method unused and possibly a hold over from when the plan was to create an agent.